src/HiveApplication.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor;
+
+use Keboola\DbExtractorLogger\Logger;
+
+class HiveApplication extends Application
+{
+    public function __construct(array $config, Logger $logger, array $state = [], string $dataDir = '/data/')
+    {
+        $config['parameters']['data_dir'] = $dataDir;
+        $config['parameters']['extractor_class'] = 'Hive';
+
+        parent::__construct($config, $logger, $state);
+    }
```

Comment:
Tady bych přidal ConfigDefinition - jako je to u MSSQL - https://github.com/keboola/db-extractor-mssql/blob/master/src/Keboola/DbExtractor/MSSQLApplication.php#L24

Reply:
Je to potrebne? Ak tam nechcem nic naviac oproti defaultnej konfiguracii?

Reply:
tady https://keboola.slack.com/archives/CQ1ASK06M/p1581497228014800 jsme se bavili že by hive mohl mít už jen ConfigRows, tak že by měl už jen `ConfigRowDefinition`

Reply:
jj. nechal som tam default config, pretoze to nicomu nevadi, ... kedze to neriesi toto repo ... ale ak to tam chcete mat explicitne, tak tam povolim iba `ConfigRowDefinition` a inak vyhodim exception.